litebird_sim/observations.py

```diff
@@ -33,18 +41,46 @@ class Observation:
             measurement unit used for `start_time`, this *must* be
             expressed in Hertz.
 
-        nsamples (int): The number of samples in this observation.
-
         use_mjd (bool): If ``True``, the value of `start_time` is
             expressed in MJD, otherwise it's in seconds.
 
+        dtype_tod (dtype): Data type of the TOD array
+
+        n_blocks_det (int): divide the detector axis of the tod in
+            `n_blocks_det` blocks
+
+        n_blocks_time (int): divide the time axis of the tod in
+            `n_blocks_time` blocks
+
+        comm: either `None` (do not use MPI) or a MPI communicator
+            object, like `mpi4py.MPI.COMM_WORLD`. Its size is required to be at
+            least `n_blocks_det` times `n_blocks_time`
+
+        root (int): rank of the process receiving the detector list, if
+            ``detectors`` is a list of dictionaries, otherwise it is ignored.
     """
 
     def __init__(
-        self, detector, start_time, sampling_rate_hz, nsamples, use_mjd=False
```

Comment:
The parameter `use_mjd` is useless in the `master` branch, as the choice to use or not astropy in computing times is done according to the type of the parameter `start_time`. It can be safely removed here.